'make deps' Not work

**Describe the bug**

At macOS, run command: `make deps`
Shown:
`make: *** No rule to make target `deps'.  Stop.`

**Reproducing the bug**

Steps to reproduce the behavior:

git clone hydra project, cd to hydra path, run command: `make deps`

**Expected behavior**

A clear and concise description of what you expected to happen.

**Environment**

- macOS Mojave Version 10.14.6 (18G3020)

--- a/README.md
+++ b/README.md
@@ -419,13 +419,6 @@ You need Go 1.13+ with `GO111MODULE=on` and (for the test suites):
 
 It is possible to develop ORY Hydra on Windows, but please be aware that all guides assume a Unix shell like bash or zsh.
 
-#### Workflows
-
-##### Install Tools
-
-When cloning ORY Hydra, run `make deps`. It will download several required dependencies. If you haven't run the command
-in a while it's probably a good idea to run it again.
-
 #### Formatting Code
 
 You can format all code using `make format`. Our CI checks if your code is properly formatted.

